CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ### 4.0.1 (Next)
 
+* [#212](https://github.com/mongoid/mongoid-rspec/pull/212): Support mongoid 7 - [@madAle](https://github.com/madAle).
```

Comment:
Nitpick: for 6 we said "Mongoid 6 support" below, my OCD tells me this should say "Mongoid 7 support" ;)

Reply:
👍